implementations/contracts/ERC725/ERC725XInit.sol

```diff
@@ -34,6 +34,38 @@ contract ERC725XInit is ERC725XCore, OwnableUpgradeable {
         uint256 _value,
         bytes calldata _data
```

Comment:
Why do we copy and paste here? we shouldn't use copy and pasted code between normal and init.